TranslationManager/Classes/Manager/TranslatableManager.swift

```diff
@@ -457,6 +467,9 @@ public class TranslatableManager<L: LanguageModel, C: LocalizationModel> {
             }
         }
 
+        //we're done with this async call, so lets leave
+        group?.leave()
+
```

Comment:
This seems to me to be called too early? Shouldn't it get called only after the the translations are set? (I think I would just put this in a defer block to make sure it's run at any point and also so we don't break this in the future if we add more code here.

Reply:
Nice catch! I've wrapped it in a `defer` now